assets/parquet/main.py

```
@@ -0,0 +1,154 @@
+        self.docker_execute("sync")
+        self.docker_execute("echo 1 >/proc/sys/vm/drop_caches", check=False, shell=True)
+        self.docker_execute('curl -X GET "http://localhost:7777/v1/operation/server/clearCache?type=memory"')
+        self.docker_execute('curl -X GET "http://localhost:7777/v1/operation/server/clearCache?type=ssd"')
+
+    def reset(self):
+        """
+        Removes a previously ingested dataset before ingesting a new one, must not throw error
+        when no dataset was ingested
+        """
+        self.docker_execute(f"mkdir -p {PARQUET_DATA_PATH}")
+        self.hive_execute(f"DELETE FROM {PARQUET_SCHEMA_NAME}.{PARQUET_TABLE_NAME} WHERE true;", check=False)
```

Ruff (0.12.2)

Possible SQL injection vector through string-based query construction

(S608)